src/commands/utils/download/download.ts

```diff
@@ -1,56 +1,50 @@
-import * as path from 'path';
-import { succeeded, Errorable, getErrorMessage} from '../errorable';
-import { Dictionary } from '../dictionary';
-import { sleep } from '../sleep';
-import * as download from 'download';
+import * as path from "path";
+import { succeeded, Errorable, getErrorMessage } from "../errorable";
+import { Dictionary } from "../dictionary";
+import { sleep } from "../sleep";
+import download from "download";
```

Comment:
I actually think this shouldn't be equivalent, but `eslint` complained that `download` was the wrong type in the previous code. I tested downloading files after the update and it still works.